HISTORY.md

```diff
@@ -14,7 +14,7 @@
 * Add `WriteBatch::Release()` that releases the batch's serialized data to the caller.
 
 ### Public API Changes
-* Add parameter `deletion_ratio` to C API `rocksdb_options_add_compact_on_deletion_collector_factory`.
+* Add C API `rocksdb_options_add_compact_on_deletion_collector_factory_del_ratio`.
```

Comment:
Added comments regarding change log. Rest LGTM. Thanks for the quick fix.

Reply:
I think it should be added as a separate entry as mentioning backward compatibility as we have already cut the branch.

Reply:
Thanks, updated the change log.